test/indexer/test_anvil.py

```diff
@@ -76,6 +97,39 @@ def test_indexing(self):
         expected_hits = self._load_canned_result(self.bundle)
         self.assertEqual(expected_hits, hits)
 
+    def test_dataset_description(self):
+        dataset_ref = EntityReference(entity_type='dataset',
+                                      entity_id='2370f948-2783-4eb6-afea-e022897f4dcf')
+        dataset_bundle = self.bundle_fqid(uuid='2370f948-2783-aeb6-afea-e022897f4dcf',
+                                          version=self.bundle.version,
+                                          entity_type=BundleEntityType.dataset)
+
+        bundles = [self.bundle, dataset_bundle]
+        for bundle_fqid in bundles:
+            bundle = cast(TDRAnvilBundle, self._load_canned_bundle(bundle_fqid))
+            bundle.links.clear()
+            bundle.entities = {dataset_ref: bundle.entities[dataset_ref]}
+            self._index_bundle(bundle, delete=False)
+
+        hits = self._get_all_hits()
+        doc_counts = defaultdict(int)
```

Comment:
```suggestion
        doc_counts: dict[DocumentType, int] = defaultdict(int)
```
Without this PyCharm is nagging when the dictionary is populated:
Unexpected type(s):(DocumentType)Possible type(s):(str)(str)

Reply:
I don't see this warning. Which code inspection setting is it associated with?

Reply:
`Incorrect type`

![Screen Shot 2023-10-30 at 4 14 58 PM](https://github.com/DataBiosphere/azul/assets/46407464/db3c762d-5d61-4687-800b-5b16fbf5560d)